lessons/modelevaluation.ipynb

```diff
@@ -0,0 +1,786 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "# Evaluating Models\n",
+    "\n",
+    "Goals:\n",
+    "\n",
+    "* Be able to design and carry out tests of model adequacy (goodness of fit), and comparisons between models\n",
+    "\n",
+    "* Understand and be prepared to use the Bayesian Evidence"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## Further reading\n",
+    "\n",
+    "* Mackay, Chapters 3 and 28\n",
+    "\n",
+    "* Gelman et al (BDA3), Chapter 6"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## Preamble\n",
+    "\n",
+    "You can't do inference without making assumptions.\n",
+    "\n",
+    "$\\longrightarrow$ We must _test_ the hypotheses defined by our models."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "Three related but distinct questions come under the heading of **model evaluation**.\n",
+    "1. Does a model describe (fit) the data well?\n",
+    "2. Does a model make accurate predictions about new data?\n",
+    "3. How probable are our models in light of the data?\n",
+    "\n",
+    "Often (2) and (3) are directly related to **model comparison** or **selection**"
```

Comment:
Perhaps clearer, therefore, to say "competing models" instead of "our models" in 3. Unless we have a way to assign an absolute probability to the model being right?

Reply:
✅